sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -347,7 +352,6 @@ export class EventGridPublisherClient {
     readonly endpointUrl: string;
     sendCloudEvents(events: SendCloudEventInput<any>[], options?: SendCloudEventsOptions): Promise<SendEventsResponse>;
     sendCustomSchemaEvents(events: Record<string, any>[], options?: SendCustomSchemaEventsOptions): Promise<SendEventsResponse>;
-    // Warning: (ae-forgotten-export) The symbol "SendEventsResponse" needs to be exported by the entry point index.d.ts
```

Comment:
This line of code change is not really related to Version Upgrade. I just found a forgotten export and added it. Please let me know if this is not exported intentionally. I can revert the changes. Thanks

Reply:
Nope, this was just me forgetting to export something.  Thank you for fixing!